p2panda-js/src/session/session.test.ts

```diff
@@ -104,8 +104,8 @@ describe('Session', () => {
         encodedEntryFixture(4).entryBytes,
         encodedEntryFixture(4).payloadBytes,
       );
-      expect(nextEntryArgs.entryHashBacklink).toEqual(
-        entryArgsFixture(5).entryHashBacklink,
+      expect(nextEntryArgs.backlink).toEqual(
```

Comment:
Something must have went wrong here, since the JSON RPC method awaits still `entryHashBacklink` and `entryHashSkiplink` params?

Reply:
Ah, right! Oops, I'll revert.

Reply:
reverted.